exporter/stackdriverexporter/config.go

```diff
@@ -20,8 +20,8 @@ import "github.com/open-telemetry/opentelemetry-collector/config/configmodels"
 type Config struct {
 	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
 	ProjectID                     string                   `mapstructure:"project"`
-	Prefix                        string                   `mapstructure:"metric-prefix"`
+	Prefix                        string                   `mapstructure:"metric_prefix"`
```

Comment:
I know that this is painful but we agreed to have everything as underscores and that is now enforced by the OTelCol